ui/src/components/virtual-scroll/QVirtualScroll.js

```diff
@@ -34,6 +34,11 @@ export default Vue.extend({
       default: () => []
     },
 
+    columnCount: {
```

Comment:
Please keep it as `columnCount: [ Number, String ],`, without a default
and move it to mixins/virtual-scroll.js